### PR TITLE
Added Chmod, Chown, Chtimes, and uid:gid to Ls

### PR DESCRIPTION
**Pull request by rwincey**
1.  Added user and group to directory listing (ls) for linux & darwin payloads. Ignored on windows & generic
2.  Added chmod, chown for linux payloads
3.  Added chtimes (timestomp) for all payloads

### CODE REVIEW

**Comment by moloch--** [comment]
dope

File: implant/sliver/handlers/handlers_linux.go [hunk 1]
```diff
@@ -60,10 +72,182 @@ var (
...
+				}
+			} else {
+				chown.Response.Err = err.Error()
+			}
+		} else {
+			chown.Response.Err = err.Error()
+		}
+	} else {
+		chown.Response.Err = err.Error()
+	}
```
Comment (rkervella) [inline implant/sliver/handlers/handlers_linux.go:249]: This whole block should be refactored to be easier to read. 90% of this doesn't need to be nested if we change the test condition to `if err != nil { //set error and return}`. We usually try to "return often, return early".

**Comment by rwincey** [comment]
Did the things

**Review by rkervella — DISMISSED** [review]

**Comment by rwincey** [comment]
Had to commit one more time to get a "Verified" commit.

**Review by rkervella — APPROVED** [review]

**Comment by rwincey** [comment]
Any clue on how to get all commits signed? Do I need to open up a whole new branch?

**Comment by rkervella** [comment]
I don't think you can retro-actively sign commits. You'd have to submit a new PR from a new branch I think. I'll force merge for this time, but please remember to sign your commits next time.
